src/MF_Modules/MFOutputShifter.cpp

```diff
@@ -0,0 +1,94 @@
+// MFOutputShifter.cpp
+//
+// Copyright (C) 2021
+
+#include "MFOutputShifter.h"
+
+MFOutputShifter::MFOutputShifter()
+{
+  _initialized = false;
+}
+
+void MFOutputShifter::setPin(uint8_t pin, uint8_t value, uint8_t refresh)
```

Comment:
What is this "refresh" thing? It doesn't get passed in anywhere that I can see and is unrelated to the naming changes.

Reply:
It is used in SetPins(); it allows to change the output values deferring the transmission once all changes have been done. This way, SetPins() can use SetPin() (instead of duplicating the code) because it otherwise it would send all data every time.

Reply:
Thanks for the explanation! This change wasn't listed in the PR description so it wasn't clear if it was intentional as part of the rename work (since it's unrelated to the two mentioned issues).

Reply:
Actually I saw it more as part of the update related to the changes in the data structure, even if not strictly. It's internal management more than anything else, but I see it's clearer now. Thanks.